doc/release/1.13.0-notes.rst

```diff
@@ -87,6 +87,11 @@ it should be regarded as a band-aid until Mingwpy is fully functional.
 
 .. _issues: https://mingwpy.github.io/issues.html
 
+Performance improvements for ``packbits`` and ``unpackbits``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Just a note for the future, I've changed the `~` to `-` for added markdown compatibility for the github releases. I'll fix these up in the release notes later.